src/sentry/api/serializers/models/group_stream.py

```diff
@@ -156,6 +160,34 @@ def get_stats(
             return self.query_tsdb(item_list, query_params, user=user, **kwargs)
 
 
+# Serializer for User Feedback
+# Used to expose the linked integration issues for a list of feedbacks (groups)
```

Comment:
```suggestion
```

- This Serializer could be used for other things besides our feedback use case
- The second line isn't quite accurate after our recent changes, since now this serializer simply takes in an ExternalIssue now

Reply:
updated the comment